### DISCUSSION TITLE
Missing email tab in opportunity

### DISCUSSION THREAD

**Discussion by pombredanne** [twentyhq/twenty#6995] [Ideas]
## Current behavior

The email tab is missing in opportunity

## Expected behavior

The email tab should be populated with any synced email with the "Point of Contact" persons.
This would enable looking at emails in context.

**Comment by FelixMalfait** [comment]
This would be great but I think we'll wait for custom layouts to do it. Because we would have to hardcode/do something dirty today to implement it. Ideally users should be able to display the mail tab on any object

**Comment by BOHEUS** [comment]
Related to https://github.com/twentyhq/core-team-issues/issues/51
